replica/mutation_dump.hh

```diff
@@ -0,0 +1,26 @@
+/*
+ * Copyright (C) 2023-present ScyllaDB
+ */
+
+/*
+ * SPDX-License-Identifier: (AGPL-3.0-or-later)
+ */
+
+#pragma once
+
+#include "db/timeout_clock.hh"
+#include "query-result.hh"
+
+namespace replica::mutation_dump {
+
+schema_ptr generate_output_schema_from_underlying_schema(schema_ptr underlying_schema);
+
+future<foreign_ptr<lw_shared_ptr<query::result>>> dump_mutations(
+        sharded<database>& db,
+        schema_ptr output_schema,
+        schema_ptr underlying_schema, // must have been generated from `output_schema`, with `generate_output_schema_from_underlying_schema()`
```

Comment:
Comment is confused, it's the output schema that must have been generated from the underlying schema.

Reply:
Fixed in v4.